yetanotherco/aligned_layer
# chore: bump up sp1 version

# Description

- This PR bumps up SP1 version from `v1.0.8-testnet` to `v1.0.1`.
- The main changes are:
  - `client.prove_compressed(&pk,stdin)` method no longer exists. To use compressed now you need to: `client.prove(&pk, stdin).compressed().run()`.
  - `client.verify_compressed(&proof, &vk)` method no longer exists. Now it's just `client.verify(&proof, &vk)`.
- The old SP1 proofs won't work with this version.

# To Test

- First, run `make deps`.
- Run `make test_sp1_go_bindings_macos` or `make test_sp1_go_bindings_linux` depending on the OS.
- Run `test_sp1_rust_ffi`.
- Run all as usual and send an SP1 proof. Make sure it's working properly.
